packages/dataviews/src/components/dataviews-filters/index.tsx

```diff
@@ -16,12 +15,12 @@ import { __, _x } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
+import { useDataViewsContext } from '../..';
+import { sanitizeOperators } from '../../utils';
+import { ALL_OPERATORS, OPERATOR_IS, OPERATOR_IS_NOT } from '../../constants';
```

Comment:
Do we need to move these lines up?

You'd hear me saying this a lot: it's important that we make the minimum viable changes to `packages/dataviews`. If we start reordering stuff around, subsequent syncs from upstream will be impossible to make.